### DISCUSSION TITLE
Cannot able deploy contract on zkSync Sepolia Testnet

### DISCUSSION THREAD

**Discussion by rxhul18** [Cyfrin/foundry-full-course-cu#2950] [General]
<img width="1020" alt="Screenshot 2024-11-30 at 1 03 01 AM" src="https://github.com/user-attachments/assets/69e9d9eb-b2a9-4ba5-b690-fb70d00493ac">
I was able to compile for normal remix sepolia but on pligin of zkSync cannot able to compile contract i tried with various version 
Any suggestion or answer?

**Comment by EngrPips** [comment]
I am glad you figured this out.
